scripts/sysman/ras.yml

```diff
@@ -40,6 +40,15 @@ etors:
       desc: "The number of errors that have occurred in caches (L1/L3/register file/shared local memory/sampler)"
     - name: DISPLAY_ERRORS
       desc: "The number of errors that have occurred in the display"
+    - name: MEMORY_ERRORS
+      desc: "The number of errors that have occurred in Memory"
+      version: "1.7"
+    - name: SCALE_ERRORS
+      desc: "The number of errors that have occurred in Scale Fabric"
+      version: "1.7"
+    - name: L3FABRIC_ERRORS
+      desc: "The number of errors that have occurred in L3 Fabric"
+      version: "1.7"
```

Comment:
S_MAX_RAS_ERROR_CATEGORY_COUNT is no longer 7, right?

what happens with ras_state_t category, which is a static array?

```
typedef struct _zes_ras_state_t
{
    zes_structure_type_t stype;                                             ///< [in] type of this structure
    const void* pNext;                                                      ///< [in][optional] must be null or a pointer to an extension-specific
                                                                            ///< structure (i.e. contains stype and pNext).
    uint64_t category[ZES_MAX_RAS_ERROR_CATEGORY_COUNT];                    ///< [in][out] Breakdown of error by category

} zes_ras_state_t;
```

Reply:
right, I'm thinking that any new error categories that are added to be made discoverable via the new extension mechanism in https://github.com/oneapi-src/level-zero-spec/pull/181 . that way we keep existing `COUNT` and avoid the issues that come with the static array here.

I can add note to that effect that new error categories would only be visible via an extension mechanism. what do you think?

Reply:
added note to programming guide with regards to this

Reply:
ZES_MAX_RAS_ERROR_CATEGORY_COUNT is tightly coupled to this. It is better to define a whole different new enum where new categories are added, and that new enum is the one used with the new extension mechanism.

Reply:
Recommend also that the new enum type aliases 1:1 the categories of the old enum type.

Reply:
sure, for enum naming - any suggestions? would ras_error_catrgories_t be fine? (it's quite specific and avoid using term "CAT" which is already causing some confusion )

Reply:
ras_error_categories_t I think it is good

Reply:
since this enum is going to be tied to the RAS extension, have added new enum to existing PR #181 . @jandres742 @wdamon-intel could you please review there? if it looks fine, we can close this PR